### DISCUSSION TITLE
How to do incremental build?

### DISCUSSION THREAD

**Discussion by ghost** [SpartanJ/ecode#199] [General]
For example, after a `git pull`, do I have to clean everything with `make -e config=release_x86_64 -C make/windows clean` or can I simply do `make -e config=release_x86_64 -C make/windows`? The compilation takes a very long time for each clean build.

**Comment by SpartanJ** [comment]
If you don't clean it will always do an incremental build, you shouldn't clean the object cache unless there's a very good reason (hint: I almost never do a clean build).
